packages/discord.js/src/structures/DMChannel.js

```diff
@@ -29,10 +31,14 @@ class DMChannel extends Channel {
 
     if (data.recipients) {
       /**
-       * The recipient on the other end of the DM
-       * @type {User}
+       * The recipient's id
+       * @type {Snowflake}
        */
-      this.recipient = this.client.users._add(data.recipients[0]);
+      this.recipientId = data.recipients[0].id;
+
+      if ('username' in data.recipients[0] || this.client.options.partials.includes(Partials.Users)) {
```

Comment:
Perhaps create a variable for `data.recipients[0]`?

Reply:
Sure! Done with c6032a6b80593ad8432556d4c701f9283ada6309.